Mathlib/Data/Nat/GCD/BigOperators.lean

```diff
@@ -30,4 +30,38 @@ theorem coprime_prod_right {ι : Type*} {x : ℕ} {s : ι → ℕ} {t : Finset 
   Finset.prod_induction s (fun y ↦ x.Coprime y) (fun a b ↦ Coprime.mul_right) (by simp)
 #align nat.coprime_prod_right Nat.coprime_prod_right
 
+theorem coprime_prod_left_iff {ι : Type*} [DecidableEq ι] {x : ℕ} {s : ι → ℕ} {t : Finset ι} :
+    Coprime (∏ i : ι in t, s i) x ↔ (∀ i : ι, i ∈ t → Coprime (s i) x) :=
+  Finset.induction_on t (by simp) fun i s his ih ↦ by
+    rw[Finset.prod_insert his, Nat.coprime_mul_iff_left, ih, Finset.forall_mem_insert]
```

Comment:
A general style comment: spaces after `rw` and `simp`:
```suggestion
    rw [Finset.prod_insert his, Nat.coprime_mul_iff_left, ih, Finset.forall_mem_insert]
```

Reply:
I corrected that notation.